### ISSUE TITLE
Conflict with Mac OS

### ISSUE THREAD

**Issue by mmattimoe** [keybase/client#21096]
[write something useful and descriptive here]

my log id: 1ef63fc515cced6f8c5ca81c

Getting an error report re available device slots. It instructs to restart but this does not fix the error

**Comment by mmattimoe** [comment]
Hi!

I am not sure what "acked" means but this is still a problem.

Here is the error:
"We were unable to load KBFS (Fuse kext). This may be due to a limitation in MacOS where there aren't any device slots available. Device slots can be taken up by apps such as VMWare, VirtualBox, anti-virus programs, VPN programs and Intel HAXM.
We were unable to load the part of Keybase that lets you access your files in your file system. You should be able to do so if you wait a few minutes and restart Keybase.

Please run `keybase log send` to report the error."

I have reported the error twice via terminal.

We need to be able to access our files and this is creating some issues. Any helpful feedback would be terrific! Thank you!

**Comment by strib** [comment]
This is a macOS limitation unfortunately. You can't run too many programs that require device slots.  If you're running some of the other programs listed in the error message, try shutting them down or uninstalling them if possible.

**Comment by strib** [comment]
You can always still access your files via the app or the `keybase fs` command line options.

**Comment by mmattimoe** [comment]
Ohhh! Gotcha. Hmm. Okay, I'll contact apple about what programs I can shut down. I am sure I have unnecessary stuff running. 

Thank you!!!

**Comment by mmattimoe** [comment]
Have a great night! Thanks for always being patient with my very minimal coding experience and knowledge. :)

**Comment by strib** [comment]
Of course!  Sorry for the confusing error message, we should fix that final sentence.  Not sure if you need to contact Apple -- you can start by looking for the programs specifically listed in those error messages and removing them if you don't need them anymore.

**Comment by mmattimoe** [comment]
Ahhh, good idea. I don't recognize them but that doesn't mean they aren't running. 

Thanks!! :)
